src/main.cpp

```diff
@@ -150,11 +150,7 @@ int main(int argc, char* argv[])
 #endif // _MSC_VER && _DEBUG
     spDebugConsole = nullptr;
 
-    // due to a Qt bug, this only safely works for both non- and HiDPI displays on 5.12+
-    // 5.6 - 5.11 make the application blow up in size on non-HiDPI displays
-#if defined (Q_OS_UNIX) && (QT_VERSION >= QT_VERSION_CHECK(5, 12, 0))
     QApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
-#endif
```

Comment:
Is this correct - you also removed the check for NOT Windows OSes - perhaps it should be:
```diff
-#if defined (Q_OS_UNIX) && (QT_VERSION >= QT_VERSION_CHECK(5, 12, 0))
+#if defined (Q_OS_UNIX)
    QApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
#endif
```

Reply:
Fixed